app/views/help/new_features.md

```diff
@@ -0,0 +1,8 @@
+<!-- When updating this file, update date to flag users of new content -->
```

Comment:
How are we able to evaluate ERB in a markdown file? I've never seen that. Can you point me to the controller for it?